### DISCUSSION TITLE
Add `check all` & `uncheck all` options for task lists

### DISCUSSION THREAD

**Discussion by Zerotask** [community/community#5490] [Projects and Issues]
Add `check all` & `uncheck all` options for task lists. This can save some time for power users and won't break the current way of interacting with task lists.

This could also be extended with the option to create an issue for every list item (instead of creating issues one-by-one).

**Comment by fdlk** [comment]
The point of our task list is a set of mental checks that the reviewer has to go through before merging a PR. A check all box would make it too tempting to skip the mental steps.

**Reply by Zerotask** [reply]
Okay, that's a valid point against a check all.

But uncheck all or an option to create issues for every line item shouldn't break your use case.
